tests/core/test_apd.py

```diff
@@ -628,3 +628,19 @@ def test_parse_time_field_00(input_, expected):
     """Ensure a time field gets parsed correctly."""
     actual = apd.parse_time_field(input_)
     assert actual == expected
+
+
+@pytest.mark.parametrize('input_,expected', (
+    ('<strong>Date:   </strong>April 18, 2019</p>', 'April 18, 2019'),
+    ('>Date:   </strong> Night of May 22 2019</p>', 'May 22, 2019'),
+    ('>Date: Night of  Oct 18 2013</p>', 'October 18, 2013'),
+    ('>Date:  night Apr 1-2012</p>', 'April 01, 2012'),
+    ('>Date:  feb 2 2019</p>', 'February 02, 2019'),
+    ('>Date:  10-1-17</p>', 'October 01, 2017'),
+    ('>Date:  Morning of 2,2,19 </p>', 'February 02, 2019'),
+    ('>Date:  3/3/19</p>', 'March 03, 2019'),
+))
+def test_parse_date_field_00(input_, expected):
+    """Ensure a date field gets parsed correctly."""
+    actual = apd.parse_date_field(input_)
+    assert actual == expected
```

Comment:
Good testing, however there is no invalid case. You should add at least one case which fails to be parsed. this would help you test the case I described in my remark about `search_dates()`.

Add a new line at the end of file.